### DISCUSSION TITLE
How to get the path

### DISCUSSION THREAD

**Discussion by HarkoTyrant** [yasirkula/UnityNativeGallery#268] [Q&A]
Hi, i'm testing this on the Unity editor and it seems to work fine but after searching everywhere i can't find where the images are been saved. 

Tried to get the path data from MediaSaveCallback but still not sure how to do it.

**Comment by yasirkula** [comment]
Yes you can retrieve saved image's path from that callback on Android but definitely check out the remarks in documentation first: https://github.com/yasirkula/UnityNativeGallery#how-to

**Comment by HarkoTyrant** [comment]
After a revision on the code i think i can't find the files because SaveImageToGallery don't save the images while running on the editor.

And still don't know how to access the values on callback. How i can see if success is true and the path? There is no example about this in the How To.

**Reply by yasirkula** [reply]
Actually TakeScreenshotAndSave uses the callback function. Notice the `Debug.Log( "Media save result: " + success + " " + path ) );` part? That's the body of the callback function.

**Reply by yasirkula** [reply]
You're right about editor functionality, it's there for preview purposes and it may not support all features.

**Reply by HarkoTyrant** [reply]
Oh yes, didn't see that line (or that there is more code at the right :P)
By the way, path return the file name too? Is because it will end with added numbers if the file already exists.

**Reply by yasirkula** [reply]
The returned path is full file path but only on Android and not in all cases (see the remarks in documentation for more info). If file exists, a unique number will be appended to its filename which will be reflected to the returned path, yeah.

**Reply by AoutoCooper** [reply]
Wow I was looking so hard to see how to use these callbacks until I saw your comment! Never had to use callbacks before, only with DOTween :)))
